### PR TITLE
Fix some style issues

### LINKED ISSUE #227: Replace Ikonograph with FontAwesome v4.7
Replace the deprecated icon library, Ikonograph, with FontAwesome v4.7.

### LINKED ISSUE #228: Sanitize some components styles
Apply the following style resets:

- **General:**

    ```css
    ol,
    ul {
      padding: 0;
    }
    ```

- **Form:**
 
    ```css
    label {
      color: var(--color-base800);
    
      &[disabled] {
        color: var(--color-base500);
      }
    }
    ```
    
    ```css
    .key-value li span:first-of-type {
      color: var(--color-base800);
    
      &[disabled] {
        color: var(--color-base500);
      }
    }
    ```

- **Accordion:**

    ```css
    .accordion-block {
      padding: 0;
    
      &.active {
        margin: 0;
      }
    }
    ```

### PR DESCRIPTION
**Pull request by giotramu**
Resolve #227 #228

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
+  "$schema": "http://json.schemastore.org/prettierrc",
```
Comment (StefanoMagrassi) [inline .prettierrc.json:2]: @giotramu this file should be renamed in `.prettierc` so as to remove this `$schema` reference

File: .prettierrc.json [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+  "arrowParens": "avoid",
   "bracketSpacing": false,
+  "endOfLine": "lf",
```
Comment (StefanoMagrassi) [inline .prettierrc.json:5]: @giotramu why this?

Comment (giotramu) [reply .prettierrc.json]: Sorry, I notice now that it's the default value. The goal was to force the Windows users to the line feed only.

**Review by StefanoMagrassi — CHANGES_REQUESTED** [review]
@giotramu with these changes it will be required to add the css import from Fontawesome CDN in html's head?

**Comment by giotramu** [comment]
> @giotramu with these changes it will be required to add the css import from Fontawesome CDN in html's head?

@StefanoMagrassi Yes, It's mandatory. I should update the README.md?

**Comment by StefanoMagrassi** [comment]
> > @giotramu with these changes it will be required to add the css import from Fontawesome CDN in html's head?
> 
> @StefanoMagrassi Yes, It's mandatory. I should update the README.md?

yep

**Comment by giotramu** [comment]
@StefanoMagrassi I have updated the documentation with some details.
It's hard for me to be more specific because the project was deprecated for a while.

**Review by StefanoMagrassi — DISMISSED** [review]

**Pull request merged**